pkg/controller/storagecluster/volumesnapshotterclasses.go

```diff
@@ -0,0 +1,114 @@
+package storagecluster
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+
+	snapapi "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SnapshotterType represents a snapshotter type
+type SnapshotterType string
+
+const (
+	rbdSnapshotter    SnapshotterType = "rbd"
+	cephfsSnapshotter SnapshotterType = "cephfs"
+)
+
+// secret name and namespace for snapshotter class
+const (
+	snapshotterSecretName      = "csi.storage.k8s.io/snapshotter-secret-name"
+	snapshotterSecretNamespace = "csi.storage.k8s.io/snapshotter-secret-namespace"
+	snapshotterNamePrefix      = "snapshotNamePrefix"
+)
+
+var (
+	snapshotterSecretNameMap = map[SnapshotterType]string{
+		rbdSnapshotter:    "rook-csi-rbd-provisioner",
+		cephfsSnapshotter: "csi-cephfs-secret",
```

Comment:
the secret name is wrong it should be `rook-csi-cephfs-provisioner`

Reply:
Thanks @Madhu-1 , changed it...